packages/vm/src/index.ts

```diff
@@ -210,32 +206,34 @@ export default class VM extends AsyncEventEmitter {
       this._common = opts.common
     } else {
       const DEFAULT_CHAIN = Chain.Mainnet
-      // const supportedHardforks = [
-      //   'chainstart',
-      //   'homestead',
-      //   'dao',
-      //   'tangerineWhistle',
-      //   'spuriousDragon',
-      //   'byzantium',
-      //   'constantinople',
-      //   'petersburg',
-      //   'istanbul',
-      //   'muirGlacier',
-      //   'berlin',
-      //   'arrowGlacier',
-      // ]
-
-      this._common = new Common({
-        chain: DEFAULT_CHAIN,
-        // supportedHardforks,
-      })
 
-      // this._supportedHardforks = supportedHardforks
+      this._common = new Common({ chain: DEFAULT_CHAIN })
     }
     this._common.on('hardforkChanged', () => {
       this._opcodes = getOpcodesForHF(this._common)
     })
 
+    this._supportedHardforks = [
+      'chainstart',
+      'homestead',
+      'dao',
+      'tangerineWhistle',
+      'spuriousDragon',
+      'byzantium',
+      'constantinople',
+      'petersburg',
+      'istanbul',
+      'muirGlacier',
+      'berlin',
+      'london',
+      'arrowGlacier',
+    ]
```

Comment:
I am not seeing any value in making this a member variable. Can we maybe stick with keeping this local? Or am I overlooking something?

Reply:
just a thought, would it be possible to use the `Hardfork` enum for the values here? The result should still be the same string values.

Reply:
@holgerd77 tbh I added it last minute, didn't really have a good reason. Reverted it to local variable

@ryanio Good call! Updated